cni-plugin/pkg/plugin/plugin.go

```diff
@@ -733,6 +733,6 @@ func Main(version string) {
 	}
 
 	skel.PluginMain(cmdAdd, nil, cmdDel,
-		cniSpecVersion.PluginSupports("0.1.0", "0.2.0", "0.3.0", "0.3.1"),
+		cniSpecVersion.PluginSupports("0.1.0", "0.2.0", "0.3.0", "0.3.1", "0.4.0", "1.0.0"),
```

Comment:
One thing I spotted, CNI version v0.4.0 has these change notes:

```
A new verb, "CHECK", was added. Runtimes can now ask plugins to verify the status of a container's attachment
A new configuration flag, disableCheck, which indicates to the runtime that configuration should not be CHECK'ed
```

We don't currently have a CHECK implementation. I suppose it would be simple enough to implement a dummy check interface that always returns "OK", or a simpler fix would be to add `disableCheck: true` to all of our CNI configs (in manifests and operator).

Reply:
I'll try to add a dummy interface.

Reply:
Added dummy function and some minor change to the default behaviour according to guideline https://www.cni.dev/docs/spec-upgrades/#general-guidance-for-all-plugins-language-agnostic